main/features/user/tasks/list/tasks-no-form.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  'use strict';
+
+  /**
+   * taskNoForm directive display a task context and is associated form.
+   * The task form and the case overview are both iframe from bonita portal
+   * The taskDetails parameters are:
+   * @param {Object}   current-task   A task object
+   * @param {boolean}  editable       true if the user can interact with the form
+   * @param {boolean}  inactive       if true, will hide tabs content
+   * @param {function} refreshAll     refresh handler, called on task submission
+   */
+
+  angular
+    .module('org.bonitasoft.features.user.tasks.details')
+    .directive('noForm', ['$window', 'taskListStore', 'userTaskAPI', 'humanTaskAPI', 'commentAPI',
+      function($window, taskListStore, userTaskAPI, humanTaskAPI, commentAPI) {
+        // Runs during compile
+        return {
+          restrict: 'AE', // E = Element, A = Attribute, C = Class, M = Comment
+          templateUrl: 'portalTemplates/user/tasks/list/tasks-no-form.html',
+          replace: false,
+          scope: {
+            currentTask: '=',
+            refreshAll: '&',
+            editable: '=',
+            inactive: '=',
+            hasForm: '='
+          },
+          link: function(scope) {
+
+            //Default inactive value to false
+            scope.inactive = scope.inactive || false;
+
+            scope.onExecuteTask = function() {
+              addComment();
```

Comment:
I don't think we should to cancel the task execution because an optional comment couldn't be added. I added a toast instead to warn the user.